AFMC-MAJCOM/task-queue
# OPAL-1010v2 - replace setup.py with pyproject.toml

This replaces the setup.py file with the pyproject.toml file. 

To test, run:
```
pip install .
```
at the root directory and make sure it installs correctly.
